backend/codesirius/problems/apps.py

```
@@ -4,3 +4,6 @@
 class ProblemsConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "problems"
+
+    def ready(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `ready`

Add return type annotation: `None`

(ANN201)